doc/whatsnew/fragments/7322.false-positive

```diff
@@ -0,0 +1,2 @@
+Improve default TypeVar name regex. Disallow names prefixed with ``T``.
+E.g. use ``AnyStrT`` instead of ``TAnyStr``.
```

Comment:
```suggestion
E.g. use ``AnyStrT`` instead of ``TAnyStr``.

Refs #7322
```

Reply:
We had a discussion recently about this, the other solution is to create an issue especially for that.